service/src/main/java/org/apache/celeborn/server/common/service/store/IServiceManager.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.server.common.service.store;
+
+import java.util.List;
+
+import org.apache.celeborn.server.common.service.config.SystemConfig;
+import org.apache.celeborn.server.common.service.config.TenantConfig;
+import org.apache.celeborn.server.common.service.model.ClusterInfo;
+
+public interface IServiceManager {
+
+  int createClusterIfNotExists(ClusterInfo clusterInfo);
```

Comment:
```suggestion
  int createCluster(ClusterInfo clusterInfo, boolean ignoreExists);
```

Reply:
> 

Currently, we have no need to createCluster without IfNotExists, we can expand this method if this is a need.

Reply:
@RexXiong, IMO, the name of this interface could be `createCluster`, no matter whether is with `IfNotExists` or `IfExists` for expanding this interface if needed.